packages/connect-web/conformance/bin/conformanceclient

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env -S npx tsx
+
+import { run } from "../client.ts";
+
+run();
```

Comment:
Can't we run the client with `tsx conformance/client.ts` straight away with `connectconformance`? That seems more direct, and we wouldn't need `conformance/bin` anymore.

Reply:
Indeed we can. Updated.